tests/functional/event_handler/test_openapi_encoders.py

```diff
@@ -184,3 +185,11 @@ def __init__(self, name: str):
 
     result = jsonable_encoder(User(name="John"))
     assert result == {"name": "John"}
+
+
+def test_openapi_encode_with_error():
+    class MyClass:
+        __slots__ = []
+
+    with pytest.raises(SerializationError, match="Unable to serializer the object*"):
```

Comment:
```suggestion
    with pytest.raises(SerializationError, match="Unable to serialize the object*"):
```

typo in exception and assertion

Reply:
Done